crates/plonk-verifier-codegen-bin/Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
-name = "codegen-bin"
+name = "plonk-verifier-codegen-bin"
```

Comment:
Could you rollback the renaming changes (here and below), please? We will rename all the crates soon, but it will be a centralized effort (otherwise it'll be hard to propagate changes through the chain of dependencies).